image/scanner/rhel/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG BASE_REGISTRY=registry.access.redhat.com
-ARG BASE_IMAGE=ubi9-minimal
-ARG BASE_TAG=9.0.0
+ARG BASE_IMAGE=ubi8-minimal
```

Comment:
ubi8/ubi-minimal

Reply:
@RTann This used to be `ubi8-minimal`, what changed from then?

Reply:
#846

Reply:
ah I forgot that was a thing, too. Both work. Let's keep what you have